### ISSUE TITLE
get device autoRotate status

### ISSUE THREAD

**Issue by assafb81** [wonday/react-native-orientation-locker#25]
Hi,
Is it possible to get device auto rotate state?
My app should be in Portrait lock always unless a video is visible.
I want to disable the option to change the orientation to Landscape when the auto rotate is off in the device level.

**Comment by wonday** [comment]
only video page can Landscape?
you can lockToPortrait() at other page and 
unlockAllOrientations() when enter video page
lockToPortrait() again when leave video page

**Comment by gp3gp3gp3** [comment]
@wonday I see this issue is closed, but I am experiencing it as a bug in this new package for Android.

In the previous `react-native-orientation` package, the event listeners would not emit the orientation changes if the phone auto-rotate was set to off by the user.  Now in this new package, it does.

This is unfortunate because I could really use [this feature](https://github.com/wonday/react-native-orientation-locker/issues/26) that has been added to this package, getting the device orientation even when we have locked the orientation via JS code.  But I also need to be able to disregard all orientation logic if the user has locked the auto-rotate status of their device.

**Comment by Jelomite** [comment]
This is a real issue. Can we reopen it?

**Comment by wonday** [comment]
merged and released to v1.0.22

**Comment by NishanthVemula** [comment]
This is not a valid solution or rather to be precise, this solution is incomplete.

Consider device is locked to portrait mode

If login is implemented in portrait mode only and the rest of the pages supports all the orientations,
Inside Login Page ComponentDidMount, `call the method to lock to portrait mode`
Inside Login Page ComponentDidUnmount, that is when moving into the next pages of the application,
```
// Inside Component Did Mount
Orientation.getAutoRotateState(s => {
    if (s) Orientation.unlockAllOrientations();
});
```
After entering the page, unlock all the orientations. there is no event emitter to listen to changes in autorotate state.

Instead of bringing the previous functionality where events are not emitted when autorotate is off, this solution provided is incomplete.
